backend/orchestrators/DefaultOrchestrator.py

```diff
@@ -67,6 +67,7 @@ def conversation_without_data(self, request_body, message_uuid):
         )
 
         history_metadata = request_body.get("history_metadata", {})
+        history_metadata = super().conversation_client.create_conversation_item(request_body, super().AZURE_OPENAI_RESOURCE, super().AZURE_OPENAI_MODEL, super().AZURE_OPENAI_TEMPERATURE, history_metadata)
```

Comment:
Same as earlier comment. Should probably be logged/captured before the call to the completions API above.